widget/index.js

```diff
@@ -55,7 +55,21 @@ var WidgetGenerator = yeoman.generators.Base.extend({
       }
       // TODO: validate not empty string?
     },
-    // TODO: checkbox prompt for 2D/3D once the WAB supports 3D
+    {
+      type: 'checkbox',
+      message: 'Is it 2D or 3D?',
+      name: 'widgetType',
+      choices: [
+        {
+          value: 'is2d',
+          name: '2D'
+        },
+        {
+          value: 'is3d',
+          name: '3D'
+        }
+      ]
+    },
```

Comment:
Initial research indicates that *for now* we should be scaffolding one or the other, so this should become a radio button prompt.